UsersAuthorizationsService/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "userauthservice",
+  "version": "1.0.0",
+  "description": "Web service aimed at filtering the requests towards proxima on the basis of the permission given by the users",
+  "main": "main.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node build/main.js"
+  },
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/node": "^9.4.7"
```

Comment:
- why don't you have express as dependencies? 
- why don't you have mongoose as dependency?
- why don't you have @types dependencies?
- we don't use ISC license, but GPL-3.0